packages/fxa-shared/metrics/amplitude.js

```diff
@@ -4,13 +4,6 @@
 
 'use strict';
 
-let APP_VERSION;
-try {
-  APP_VERSION = /^[0-9]+\.([0-9]+)\./.exec(
-    require('../../../package.json').version
```

Comment:
This `require` statement used to assume we were being invoked from inside the `node_modules` directory. Now that's not the case, we needed another way to access the version number of the consumer. Hence passing it in as data below, and the equivalent changes in `fxa-content-server` and `fxa-auth-server`.